docs/resources/mapreduce_cluster.md

```diff
@@ -0,0 +1,354 @@
+  nodes (ECSs).
+  If this parameter is not empty, the master node will use the password.
+  Changing this will create a new MapReduce cluster resource.
+
+* `safe_mode` - (Optional, Bool, ForceNew) Specifies whether the running mode of the MapReduce cluster is secure,
+  default to true.
+  - true: enable Kerberos authentication.
+  - false: disable Kerberos authentication.
+  Changing this will create a new MapReduce cluster resource.
+
+* `security_group_ids` - (Optional, List, ForceNew) Specifies an array of one or more security group ID to attach to
+  the MapReduce cluster.
+  If using the specified security group, the group need to open the specified port (9022) rules.
+
+* `analysis_core_nodes` - (Optional, List) Specifies a list of the informations about the analysis core nodes in the MapReduce cluster.
+  The `nodes` object structure of the `analysis_core_nodes` is documented below.
+
+* `streaming_core_nodes` - (Optional, List) Specifies a list of the informations about the streaming core nodes in the MapReduce cluster.
+  The `nodes` object structure of the `streaming_core_nodes` is documented below.
+
+* `analysis_task_nodes` - (Optional, List) Specifies a list of the informations about the analysis task nodes in the MapReduce cluster.
+  The `nodes` object structure of the `analysis_task_nodes` is documented below.
+
+* `streaming_task_nodes` - (Optional, List) Specifies a list of the informations about the streaming task nodes in the MapReduce cluster.
+  The `nodes` object structure of the `streaming_task_nodes` is documented below.
+
+* `tags` - (Optional, Map, ForceNew) Specifies the key/value pairs to associate with the cluster.
+
+The `nodes` block supports:
+
+* `flavor` - (Required, String, ForceNew) Specifies the instance specifications for each nodes in node group.
+  Changing this will create a new MapReduce cluster resource.
+
+* `node_number` - (Required, Int) Specifies the number of nodes for the node group.
+
+  -> **NOTE:** Only the core group and task group updations are allowed.
+
+  The number of nodes after scaling cannot be less than the number of nodes originally created.
+
+* `root_volume_type` - (Required, String, ForceNew) Specifies the system disk flavor of the nodes.
+  Changing this will create a new MapReduce cluster resource.
+
+* `root_volume_size` - (Required, Int, ForceNew) Specifies the system disk size of the nodes.
+  Changing this will create a new MapReduce cluster resource.
+
+* `data_volume_type` - (Optional, String, ForceNew) Specifies the data disk flavor of the nodes.
+  Required if `data_volume_count` is not empty.
+  Changing this will create a new MapReduce cluster resource.
+
+* `data_volume_size` - (Optional, Int, ForceNew) Specifies the data disk size of the nodes.
+  Required if `data_volume_count` is not empty.
+  Changing this will create a new MapReduce cluster resource.
+
+* `data_volume_count` - (Optional, Int, ForceNew) Specifies the data disk number of the nodes.
```

Comment:
please add more description of the parameter.

Reply:
The limit of the `data_volume_count` for each node groups has been supported.